lib/Config.ts

```diff
@@ -93,6 +95,13 @@ class Config {
       }
     }
 
+    const grpcCert = args.grpc ? args.grpc.certPath : this.config.grpc.certPath;
+    const grpcKey = args.grpc ?  args.grpc.keyPath : this.config.grpc.keyPath;
+
+    if (!fs.existsSync(grpcCert) && !fs.existsSync(grpcKey)) {
+      generateCertificate(grpcCert, grpcKey);
```

Comment:
I would prefer generating the certificate in `GrpcServer.ts`. There is already enough stuff happening in the config. Also I don't think that `generateCertificate` should be in `Utils.ts` unless we are planning to use it more that once.

Reply:
@michael1011 I think it would be better in `Config.ts` it makes the most sense to me that way, I moved some functions to `Utils.ts` any way because they are used in `Command.ts` as well.